tailwind.config.js

```diff
@@ -0,0 +1,169 @@
+module.exports = {
+  theme: {
+    backgroundOpacity: {
```

Comment:
We should be able to remove the `backgroundOpacity`, `opacity`, and `spacing` keys and, in turn, inherit these values from the [default config](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js).